### DISCUSSION TITLE
3dtile has gaps

### DISCUSSION THREAD

**Discussion by hahaxiaowai** [visgl/deck.gl#5301] [Q&A]
[example](https://codesandbox.io/s/frosty-shadow-97g0n?file=/src/index.js)
I use Tile3DLayer to add 3dtile exposed by cesium,but the tile boundaries do not completely coincide,what caused this?

**Comment by Pessimistress** [comment]
See https://github.com/visgl/deck.gl/issues/4262

**Reply by ibgreen** [reply]
Linked the corresponding loaders.gl issue to the deck.gl issue. https://github.com/visgl/loaders.gl/issues/642. 

The loaders.gl folks can take another look at this in the background but it will likely take some time to make progress.

**Reply by hahaxiaowai** [reply]
Well, there seems to be no existing solution. Thanks for answering ：）
